docs/proposals/test-reports/Smart Coding benchmark suite Proposal_zh.md

```diff
@@ -0,0 +1,129 @@
+# 背景
+大型语言模型（LLM）在代码生成、自动编程、代码分析等任务中展现出了强大的能力，但这些模型通常是在通用代码数据上训练的，往往不能充分利用实际场景中软件工程师的协作和反馈。为了构建更加智能高效的代码生态，需要建立协作代码数据集和评测基准，促进LLM与软件工程师的紧密协作。本项目旨在基于开源边缘计算框架KubeEdge-Ianvs构建LLM协作代码智能体对齐数据集和评测基准。该数据集将包括软件工程师在开发过程中的行为轨迹、反馈和迭代过程，以及相关的代码版本和注释信息。通过这些数据，我们将设计评测指标和基准来衡量LLM在代码生成、推荐和分析等任务中的表现，促进LLM与软件工程师之间的协作。
```

Comment:
The proposals are not needed in this implementation PR. Shall keep the proposal in the proposal PR, i.e., https://github.com/kubeedge/ianvs/pull/120

Besides, all the content written in Chinese should be translated into English to facilitate international understanding and usage.